docs/devguide/repl.rst

```diff
@@ -4,42 +4,90 @@
 Accessing the REPL
 ==================
 
-Accessing the REPL on the micro:bit requires:
+REPL (Read-Evaluate-Print-Loop) allows the micro:bit to read and evaluate code 
+in real-time as you write it.
 
-    * Using a serial communication program
-    * Determining the communication port identifier for the micro:bit
-    * Establishing communication with the correct settings for your computer
+Accessing the REPL on the micro:bit will require you to:
 
-If you are a Windows user you'll need to install the correct drivers. The
+    * Determine the communication port identifier for the micro:bit
+    * Use a program to establish communication with the device
```

Comment:
The additional indentation here will format this as code, so it should probably be removed:

![image](https://user-images.githubusercontent.com/29712657/46605909-05ebc000-caf3-11e8-948f-40e8dccab628.png)